{"name": "Авторизация существующего пользователя с неверным email", "status": "passed", "description": "Тест проверяет появление ошибки,при попытке авторизоваться с неверным email", "start": 1724065596050, "stop": 1724065596290, "uuid": "bf5096f2-09c8-474d-8f7a-de332ad1d520", "historyId": "0e9b7e9698dbc63b945ffce965e0c30b", "testCaseId": "0e9b7e9698dbc63b945ffce965e0c30b", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_incorrect_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-L3DLL49"}, {"name": "thread", "value": "13308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}